### PR TITLE
Add a theme colour meta tag

### PR DESCRIPTION
**Pull request by gmarty**
- Add a theme colour meta tag
- Minor linting

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,6 +17,11 @@
       return;
     }
 
+    var themeColourMetaTag = document.createElement('meta');
+    themeColourMetaTag.setAttribute('name', 'theme-color');
+    themeColourMetaTag.setAttribute('content', TWITTER_COLOUR);
```
Comment (etiennesegonzac) [inline index.js:22]: so british :)

**Comment by etiennesegonzac** [comment]
Thanks! Still have a few things to fix before submitting it to the hacker place but this is a great addition!

File: index.js (hunk 1 above)
Comment (gmarty) [inline index.js:22]: It's became a second nature now :-)

**Comment by gmarty** [comment]
I've noticed a couple of bugs. Do you want me to open issues in here?
@cwiiis is also using it.

**Comment by etiennesegonzac** [comment]
Yes! Let's use the github issues.

**Pull request merged**